edx_django_utils/monitoring/README.rst

```diff
@@ -1,10 +1,19 @@
 Monitoring Utils
 ================
 
-Includes middleware and utility methods for adding custom metrics and for
-better monitoring memory consumption.
+This is our primary abstraction from 3rd party monitoring libraries such as newrelic.agent. It includes middleware and utility methods for adding custom metrics and for better monitoring memory consumption.
 
-Here is how you add the middleware::
+If, for some reason, you need low level access to the newrelic agent, please extend this library to implement the feature that you want. Applications should never include ``import newrelic.agent`` directly.
```

Comment:
Non-blocking: Do we have any linting to prevent this?  Just curious if this is some future work we want to do.

Reply:
@feanil: Not that I am aware of.  This is also a place where I'd like for myself, and possibly the team, to know how to add linting rules for things like this very quickly.  I feel like I've heard us say "we should add linting for..." many times, and I don't think we ever do.  I see a "how-to" in someone's future.